cpp/README

```diff
@@ -263,6 +263,20 @@ Troubleshooting CMake via ccmake
 
 Building the library on Windows (Visual Studio)
 -----------------------------------------------
+We recommend that libphonenumber on Windows be built with Visual Studio 2015
+Update 2 (https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update2-vs)
+or later.
+
+This enables support of plain UTF-8 source files, for which you also need to specify
+the /utf-8 option in build files; see "New Options in VS2015 Update 2" at
+https://blogs.msdn.microsoft.com/vcblog/2016/02/22/new-options-for-managing-character-sets-in-the-microsoft-cc-compiler/,
+or a Chrome patch for an example:
+https://codereview.chromium.org/2488853002/diff/80001/build/config/compiler/BUILD.gn
+
+Without this version and the option, your compilation will likely break when
```

Comment:
We only have them in comments, because we made it explicitly not break, so I would say "your compilation will have warnings when ...."
From what I remember, we also made the comments the // kind -> /* style risks breakage because you need the */ to be correctly detected.

Unless there are files elsewhere where it matters, e.g. in the geocoding source code which was contributed so I don't know so well, or we changed policy at some point?